src/sections/peers.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import { useEffect } from 'react';
+import Section from '../future-hopr-lib-components/Section';
+import { useAppDispatch, useAppSelector } from '../store';
+import { actionsAsync } from '../store/slices/node/actionsAsync';
+
+function PeersPage() {
+  const dispatch = useAppDispatch();
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
+  const peers = useAppSelector((selector) => selector.sdk.peers);
+  const aliases = useAppSelector((selector) => selector.sdk.aliases);
+
+  useEffect(() => {
+    dispatch(
+      actionsAsync.getPeersThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
+    dispatch(
+      actionsAsync.getAliasesThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+      })
+    );
```

Comment:
```ts
if (loginData) {
   handleRefresh();
}
```

Reply:
Regarding this, in a different PR, this was implemented in a similar way and in the Review it was adviced to remove it:
https://github.com/hoprnet/hopr-admin/pull/40#discussion_r1221068542